.github/workflows/pypi_upload.yml

```diff
@@ -68,9 +68,10 @@ jobs:
       - name: generate matrix (PR)
         if: github.event_name == 'pull_request'
         run: |
-          cibuildwheel --print-build-identifiers --platform linux \
-          | pyp 'json.dumps({"only": x, "os": "ubuntu-latest"})' \
-          | pyp 'json.dumps(list(map(json.loads, lines)))' > /tmp/matrix
+          {
```

Comment:
Does this change anything? I'd expect it to work the same as before.

Reply:
No change, I was just messing with this logic to give me macOS wheel builds I could test (linked above)